.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,64 @@
+name: CI Build
+
+on: [push]
+
+jobs:
+  test:
+    name: Tests
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macOS-latest]
+
+    steps:
+      - if: matrix.os == 'macOS-latest'
+        run: |
+          brew update
+          brew cask install google-chrome
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 8.x
+      - name: install yarn
+        run: npm install -g yarn
```

Comment:
yarn is being preinstalled from the `setup-node` action. this step is not needed most of the time

Reply:
Removed explicit installation of yarn. Thanks!